src/main/java/com/TableTalk/Enterprise/dto/Game.java

```diff
@@ -5,13 +5,9 @@
 import lombok.Data;
 
 public @Data class Game {
-    private String id;
+    private String gameId;
```

Comment:
This is likely OK with just "id", because the class name, "Game" gives it the context of game ID.